account_invoice_line_salesperson/views/account_invoice_views.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+    <record id="invoice_form_inherit" model="ir.ui.view">
+        <field name="name">account.invoice.form.inherit</field>
+        <field name="model">account.invoice</field>
+        <field name="inherit_id" ref="account.invoice_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='invoice_line_ids']//tree//field[@name='name']" position="after">
+                <field name="invoice_type" invisible="1" />
+                <field name="user_id" attrs="{'invisible': [('invoice_type', 'not in', ['out_invoice', 'out_refund'])]}"/>
```

Comment:
I think it's redundant to consider invoice_type here.  `account.invoice_form` is only used for out_invoice and out_refund, IIUC.

Reply:
My mistake. I thought they shared the same view like another versions.